packages/ibm-products/src/components/Checklist/Checklist.mdx

```diff
@@ -0,0 +1,43 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
```

Comment:
We changed the way docs are handled since the migration to Storybook 7. See one of the [other components](https://github.com/carbon-design-system/ibm-products/blob/main/packages/ibm-products/src/components/CreateTearsheet/CreateTearsheet.docs-page.js) as an example.